batch_perf/results.txt

```diff
@@ -0,0 +1,35 @@
+On a 8 core 16MB - Apple MBP El Capitan
+
+                                                                             user     system      total        real
+redis input batch: 1000, lrange, ltrim, size: 5000, slow: 0ms            1.700000   0.070000   1.770000 (  0.362000)
+redis input batch: 250, lrange, ltrim, size: 5000, slow: 0ms             1.670000   0.080000   1.750000 (  0.363000)
+redis input batch: 125, lrange, ltrim, size: 5000, slow: 0ms             1.690000   0.070000   1.760000 (  0.364000)
+redis input batch: 100, lrange, ltrim, size: 5000, slow: 0ms             1.750000   0.070000   1.820000 (  0.369000)
+redis input batch: 50, lrange, ltrim, size: 5000, slow: 0ms              1.790000   0.090000   1.880000 (  0.399000)
+redis input batch: 10, lrange, ltrim, size: 5000, slow: 0ms              2.270000   0.110000   2.380000 (  0.541000)
+redis input batch: 1, blpop, size: 5000, slow: 0ms                       4.490000   0.390000   4.880000 (  1.496000)
+
+redis input batch: 1000, old loop, size: 5000, slow: 0ms                 1.750000   0.090000   1.840000 (  0.391000)
```

Comment:
I think there is a big difference - the amount of LUA that executes is significantly less.